h/presenters/group_membership_json.py

```diff
@@ -4,6 +4,7 @@
     EditGroupMembershipContext,
     GroupMembershipContext,
 )
+from h.util.datetime import utc_iso8601
```

Comment:
This handles None, no problem: 

https://github.com/hypothesis/h/blob/main/h/util/datetime.py#L8